### PR TITLE
Fix new pylint 1.5.0 violations

### PR DESCRIPTION
**Pull request by bdero**

### CODE REVIEW

**Comment by bdero** [comment]
Fun stuff. ;)
screen

File: roles/tests/test_user_models.py [hunk 1]
```diff
@@ -49,8 +49,8 @@ class FooObject(object):
             def __init__(self, foo_var):
                 self.foo_var = foo_var
 
-        user_group2 = FooObject('bar')
-        self.assertFalse(user_group1.__eq__(user_group2))
+        foo_object = FooObject('bar')
```
Comment (noisecapella) [inline roles/tests/test_user_models.py:52]: Why is this change necessary? Just a better name?

**Comment by noisecapella** [comment]
I left a comment but it's not important :+1:

File: roles/tests/test_user_models.py (hunk 1 above)
Comment (bdero) [inline roles/tests/test_user_models.py:52]: pylint now errors when it sees a variable reassignment with a type different than the initial assignment type. Also, I couldn't name it `baz` because that's a "forbidden identifier". :)

**Comment by bdero** [comment]
:8ball:
